crates/hotshot/src/traits/election/static_committee.rs

```diff
@@ -19,37 +18,48 @@ use hotshot_types::{
 use utils::anytrace::Result;
 
 #[derive(Clone, Debug, Eq, PartialEq, Hash)]
-
 /// The static committee election
 pub struct StaticCommittee<T: NodeType> {
     /// The nodes eligible for leadership.
     /// NOTE: This is currently a hack because the DA leader needs to be the quorum
     /// leader but without voting rights.
     eligible_leaders: Vec<<T::SignatureKey as SignatureKey>::StakeTableEntry>,
+    /// The nodes eligible for leadership.
+    da_eligible_leaders: Vec<<T::SignatureKey as SignatureKey>::StakeTableEntry>,
 
```

Comment:
Shouldn't the `da_eligible_leaders` be == `eligible_leaders` since they are the same for now? In that case, `da_leader()` or associated functions should always return as a leader from the selection of all nodes, not just DA

Reply:
https://github.com/EspressoSystems/HotShot/pull/3867/commits/36f2f13005b314f9dbfbc2b489a81ec8da9b1933